mlflow/protos/unity_catalog_oss_messages.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto2";
+
+package mlflow.unitycatalog;
+
+import "scalapb/scalapb.proto";
+import "databricks.proto";
+
+option java_package = "com.databricks.api.proto.managedcatalog";
+option java_generate_equals_and_hash = true;
+option (scalapb.options) = {
+  flat_package: true,
+};
+
+// Registered Model Management API
+
+message CreateRegisteredModel {
+  optional RegisteredModelInfo registered_model_info = 1;
+
+  message Response {
+    optional RegisteredModelInfo registered_model_info = 1;
+  }
+}
+
+message RegisteredModelInfo {
+  // Input fields:
+
+  // [Create:REQ Update:OPT] Name of Model, relative to parent Schema.
+  optional string name = 1;
+  // [Create:REQ Update:IGN] Name of parent Catalog.
+  optional string catalog_name = 2;
+  // [Create:REQ Update:IGN] Name of parent Schema relative to its parent Catalog.
+  optional string schema_name = 3;
+  // [Create:IGN Update:OPT] Username of current owner of Model.
+  optional string owner = 4;
+  // [Create,Update:OPT] User-provided free-form text description.
+  optional string comment = 5;
+  // [Create:OPT] Registered model tags
+  repeated TagKeyValue tags = 20;
```

Comment:
this is probably still correct syntax-wise since the number 20 is unique, but why are we putting 20 here? (copy paste?)

Reply:
Correct, it's a copy-paste from Databricks UC protos.